src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -730,4 +730,10 @@ Do you want to proceed?</value>
   <data name="TheRenameTrackingSessionWasCancelledAndIsNoLongerAvailable" xml:space="preserve">
     <value>The rename tracking session was cancelled and is no longer available.</value>
   </data>
+  <data name="Rename2" xml:space="preserve">
+    <value>Rename '{0}'</value>
+  </data>
+  <data name="SingleQuote" xml:space="preserve">
```

Comment:
If we truncated the identifier by character count, what's the worst that could happen? Is there a language with characters much wider than "W"? We could just truncate to length and accept that the dashboard might expand slightly (since the alternative would be visually clipping something).